### DISCUSSION TITLE
HFCs and PFCs unspecified - what is unit?

### DISCUSSION THREAD

**Discussion by TheoC-ERM** [USEPA/supply-chain-factors#41] [Q&A] [answered]
Hi everyone, 

Since the newly released US EEIO Supply Chain GHG Emission Factors were calculated using Global Warming Potentials (GWP) from the 4th IPCC Assessment Report, it would be interesting to update these EFs using the most up to date GWP values from the 6th Assessment Report. 

Therefore, I was wondering what Global Warming Potential was used for “unspecified HFCs and PFCs” (see table below), and why these values were used?

Moreover, I tried to back-calculate the GWP for unspecified HFCs & PFCs by using the below GWP (column AR4) and the data from the csv file entitled “SupplyChainGHGEmissionFactors_v1.2_NAICS_byGHG_USD2021.csv”, containing the kg of each unique GHG emitted per USD per commodity. 

I thought that by multiplying the emission factor for each unique GHG with the relevant GWP, summing up the resulting CO2e emission factors for each unique GHG, and subsequently deducting the resulting value (i.e., cumulative CO2e EF) from the carbon dioxide equivalents (CO2e) per USD indicator (file SupplyChainGHGEmissionFactors_v1.2_NAICS_CO2e_USD2021.csv), I would be left with the CO2e relative to “unspecified HFCs & PFCs”.

However, this was not conclusive, as for some commodities, the sum of the CO2e emission factors for each unique GHG (excluding unspecified HFCs & PFCs) is superior to the global CO2e emission factors provided in this dataset: SupplyChainGHGEmissionFactors_v1.2_NAICS_CO2e_USD2021.csv (e.g., for Air and Gas Compressor Manufacturing)

Thus, I was wondering if there is a rational explanation for that, and if we could be provided with the GWP values used for each unique GHG, or a piece of code (R) allowing to use the existing model and extract these values (e.g., a table similar to the “table_by_gas” resulting from the R code provided, but with CO2e emission factors instead)   

Thanks in advance for your help :)

Theo 

![image](https://user-images.githubusercontent.com/131887787/235496599-8c88965b-c0e3-4faa-b575-df21f04f58f7.png)

**Comment by bl-young** [comment, marked as answer]
> Therefore, I was wondering what Global Warming Potential was used for “unspecified HFCs and PFCs” (see table below), and why these values were used?

There are a quantity of emissions for which we are not able to identify the species for these HFCs and PFCs, from the original inventory source (EPA's Greenhouse Gas Inventory). As a result, they are maintained in their source units. These flows should be indicated with units of **kg CO2 eq.** (not kg as shown in the csv file). That is, they are already characterized using AR4.

Because they are unspecified, we can not ascertain what, if any, difference in characterized impact those flows might represent under AR5 or AR6.

**Reply by Brian-Solomon** [reply]
So you are saying the current entries for "HFCs and PFCs, unspecified" with "kg/2021 USD, purchaser price" is actually _**incorrect**_, that the units here are actually kg CO2e based on AR4?

I'm referring to this file: SupplyChainGHGEmissionFactors_v1.2_NAICS_byGHG_USD2021.csv

Thank you.

**Reply by bl-young** [reply]
Yes, that is correct. There is a pending fix that will correct the generation of those csv files.
see PR https://github.com/USEPA/supply-chain-factors/pull/44

**Comment by WesIngwersen** [comment]
Unit error is resolved with [v1.2.2 release](https://github.com/USEPA/supply-chain-factors/releases/tag/v1.2.2).
